makeflow/src/mountfile.c

```diff
@@ -0,0 +1,660 @@
+/*
+Copyright (C) 2008- The University of Notre Dame
```

Comment:
In the future we may want to move this to dttools, as we can add similar functionality to other tools (e.g., a wq master).